tee-worker/app-libs/stf/src/trusted_call.rs

```diff
@@ -44,38 +58,36 @@ pub use litentry_primitives::{
 	Web3Network,
 };
 use log::*;
-use sp_core::crypto::AccountId32;
+use sp_core::{
+	crypto::{AccountId32, UncheckedFrom},
+	ed25519,
+};
 use sp_io::hashing::blake2_256;
 use sp_runtime::MultiAddress;
 use std::{format, prelude::v1::*, sync::Arc};
 
-#[cfg(feature = "evm")]
-use ita_sgx_runtime::{AddressMapping, HashedAddressMapping};
-use itp_node_api::metadata::NodeMetadataTrait;
-
-#[cfg(feature = "evm")]
-use crate::evm_helpers::{create_code_hash, evm_create2_address, evm_create_address};
-
 pub type IMTCall = ita_sgx_runtime::IdentityManagementCall<Runtime>;
 pub type IMT = ita_sgx_runtime::pallet_imt::Pallet<Runtime>;
 
 #[derive(Encode, Decode, Clone, Debug, PartialEq, Eq)]
 #[allow(non_camel_case_types)]
 pub enum TrustedCall {
 	#[codec(index = 0)]
-	balance_set_balance(Identity, AccountId, Balance, Balance),
+	noop(Identity),
```

Comment:
Can we skip this TrustedCall or add it after `send_erroneous_parentchain_call` with `index=18`?

Reply:
Do we want to let users dispatch such calls ? If not maybe we can skip them ?

Reply:
@kziemianek do you mean dispatching integritee calls? Unlikely.
They are kept untouched to not make "too much" code difference which hopefully facilitate the maintenance. If we follow this principle (= remove code that we don't need), then a lot of code will be removed 😂

Reply:
@Kailai-Wang yes, but I was thinking that we can make this trade off and ignore not needed changes in enums that are part of public contract.

Using offseted indices sounds good to me too.

Reply:
I'll use offset indices for now.

Just had a rough check - if we remove these enums, then quite some code will be affected including the imported fns/crates.. that would probably result in a larger change scope :(